homeassistant/components/recorder/statistics.py

```diff
@@ -125,22 +125,22 @@
 
 
 STATISTIC_UNIT_TO_UNIT_CONVERTER: dict[str | None, type[BaseUnitConverter]] = {
-    DistanceConverter.NORMALIZED_UNIT: DistanceConverter,
-    EnergyConverter.NORMALIZED_UNIT: EnergyConverter,
-    MassConverter.NORMALIZED_UNIT: MassConverter,
-    PowerConverter.NORMALIZED_UNIT: PowerConverter,
-    PressureConverter.NORMALIZED_UNIT: PressureConverter,
-    SpeedConverter.NORMALIZED_UNIT: SpeedConverter,
-    TemperatureConverter.NORMALIZED_UNIT: TemperatureConverter,
-    VolumeConverter.NORMALIZED_UNIT: VolumeConverter,
+    **{unit: DistanceConverter for unit in DistanceConverter.VALID_UNITS},
```

Comment:
Do we really need to create this lookup dict. Can't we just look up units as we go?

Reply:
We could, if we want it to be slower I guess 🤔